ui/components/src/ui-components/propertygrid/dataproviders/FilteringDataProvider.ts

```diff
@@ -5,132 +5,212 @@
 /** @packageDocumentation
  * @module PropertyGrid
  */
-
+import { IDisposable } from "@bentley/bentleyjs-core";
 import { PropertyRecord, PropertyValue, PropertyValueFormat } from "@bentley/ui-abstract";
+import { PropertyRecordMatchInfo } from "../../../ui-components";
```

Comment:
Should not import from barrel file of the same repo

Reply:
Changed